presidio-analyzer/analyzer/app_tracer.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+
+class AppTracer:
+    """This class handles app traces"""
+    def __init__(self, enable_interpretability=True):
+
```

Comment:
enable_interpretability controls trace output. i would remove interpretability from the name.